updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc

```diff
@@ -12,22 +12,49 @@ WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to
+In these scenarios, you can create multiple custom machine config pools (MCPs) to prevent certain worker nodes from updating when you update the cluster.
+After the rest of the cluster is updated, you can update those worker nodes in batches at appropriate times.
 
-During your first maintenance window, you would pause the MCP for *workerpool-A*, *workerpool-B*, and *workerpool-C*, then initiate the cluster update. This updates components that run on top of {product-title} and the 10 nodes which are members of the *workerpool-canary* MCP, because that pool was not paused. The other three MCPs are not updated, because they were paused.  If for some reason, you determine that your cluster or workload health was negatively affected by the *workerpool-canary* update, you would then cordon and drain all nodes in that pool while still maintaining sufficient capacity until you have diagnosed the problem. When everything is working as expected, you would then evaluate the cluster and workload health before deciding to unpause, and thus update, *workerpool-A*, *workerpool-B*, and *workerpool-C* in succession during each additional maintenance window.
+[id="example_{context}"]
+== Example Canary update strategy
 
-While managing worker node updates using custom MCPs provides flexibility, it can be a time-consuming process that requires you execute multiple commands. This complexity can result in errors that can affect the entire cluster. It is recommended that you carefully consider your organizational needs and carefully plan the implemention of the process before you start.
+The following example describes a canary update strategy where you have a cluster with 100 nodes with 10% excess capacity, you have maintenance windows that must not exceed 4 hours, and you know that it takes no longer than 8 minutes to drain and reboot a worker node.
 
 [NOTE]
 ====
-It is not recommended to update the MCPs to different {product-title} versions. For example, do not update one MCP from 4.y.10 to 4.y.11 and another to 4.y.12.
-This scenario has not been tested and might result in an undefined cluster state.
+The previous values are an example only.
+The time it takes to drain a node might vary depending on factors such as workloads.
 ====
 
+.Defining custom machine config pools
+
+In order to organize the worker node updates into separate stages, you could begin by defining the following MCPs:
+
+* *workerpool-canary* with 10 nodes
+* *workerpool-A* with 30 nodes
+* *workerpool-B* with 30 nodes
+* *workerpool-C* with 30 nodes
+
+.Updating the canary worker pool
```

Comment:
I can do that. My main intention was to segment these chunks of content with some sort of formatted text, but without over-saturating the right-side nav of the assembly. So I can make these level 2 subheadings, but I'll make them [discrete] since they're not necessary to include in the navigation.

Reply:
Someone mentioned you might make them discrete, then said: 

I agree, those should be L2s. If the author is trying to make the TOC uncluttered, they could make them [discrete], but I think it's better for the user to have those in the TOC anyway.

So in a merge review you might get pushback for it. I don't have a huge preference; I think the ToC could maybe _look_ better with them. up to you.

Reply:
I have a slight preference to keep them discrete so I'll see what the merge reviewer says. If they need to have the discrete attribute removed then that would be fine with me as well.